src/Umbraco.Web.UI.Client/src/packages/search/examine-management-dashboard/views/section-view-examine-searchers.ts

```diff
@@ -212,7 +239,19 @@ export class UmbDashboardExamineSearcherElement extends UmbLitElement {
 					</uui-tag>
 				</uui-icon-registry-essential>
 			</button>
-		</div>`;
+		</div>
+		<div style="resize: horizontal; overflow: hidden; padding: 6px;">
+			<uui-pagination 
+			.total=${this._totalPages}
+			.current=${this._currentPage}
+			firstlabel=${this.localize.term('general_first')}
+            previouslabel=${this.localize.term('general_previous')}
+            nextlabel=${this.localize.term('general_next')}
+            lastlabel=${this.localize.term('general_last')}	
+			@change=${this.#onPageChange}
+			}></uui-pagination>
```

Comment:
Indentation is a little off here, and it looks like there's an extra trailing `}` on the last line.